cms/cms_toolbars.py

```diff
@@ -430,11 +430,11 @@ def change_language_menu(self):
                     add_plugins_menu.add_modal_item(name, url=url)
 
             if remove:
-                translation_delete_url = admin_reverse('cms_pagecontent_delete', args=(self.title.pk,))
-
                 remove_plugins_menu = language_menu.get_or_create_menu('{0}-del'.format(LANGUAGE_MENU_IDENTIFIER), _('Delete Translation'))
                 disabled = len(remove) == 1
                 for code, name in remove:
+                    pagecontent = self.title.page.get_title_obj(code)
```

Comment:
This is very strange, self.title is the title object

Reply:
Ah, we got `self.page` as well, so it would make more sense to use that.

Reply:
Would this need to use the get_admin_title method to be able to get the drafts being as any delete or copy commands will be executed by an admin user only?

Reply:
I would say yes, it would indeed need to use `get_admin_title_obj` to be able to access drafts.